doc/replication.md

```diff
@@ -305,8 +290,9 @@ Replicator replicator = ReplicatorFactory.oneway(pullReplication);
 The following APIs are still supported but deprecated. They will be soon removed from the library.
```

Comment:
Would be good to note by each class/API when it was deprecated (when we know that!)